org.opent2t.sample.lamp.superpopular/com.smartthings.lightbulb/js/tests/unitTests.js

```diff
@@ -2,13 +2,15 @@ var test = require('ava');
 var translatorPath = require('path').join(__dirname, '..');
 var runLampTests = require('opent2t-device-lamp/lampTests');
 var deviceData = require('./devicedata');
+var testData = require('./testdata');
 var MockHub = require('opent2t-device-smartthingshub/mockSmartthingsHub');
 var mockHub = new MockHub(deviceData);
 
 var settings = {
```

Comment:
>var [](start = 0, length = 3)

I know in TS in the opent2t-client repo Jason had me run 'npm run lint' and it actually caught syntax/spacing/disallowed 'var' usage (to switch to let instead) -- what would it take to enable such rules as part of our verifiers?